src/components/ListItem.js

```diff
@@ -1,14 +1,32 @@
 import React from "react";
-import { View, StyleSheet, Text, Dimensions, Image } from "react-native";
+import {
+  View,
+  StyleSheet,
+  Text,
+  Dimensions,
+  Image,
+  TouchableHighlight,
+} from "react-native";
 
 const ListItem = (props) => {
   return (
     <View>
-      <Image
-        source={{ uri: props.article.image }}
-        style={styles.image}
-        testID={`article-image-${props.article.id}`}
-      />
+      <TouchableHighlight
+        testID={`article-${props.article.id}`}
+        key={props.article.id}
+        underlayColor="white"
+        onPress={() => {
+          props.navigation.navigate("Back", {
+            articleId: props.article.id,
+          });
+        }}
+      >
+        <Image
+          source={{ uri: props.article.image }}
+          style={styles.image}
+          testID={`article-image-${props.article.id}`}
+        />
```

Comment:
Shouldn't we have the TouchableHighlight wrapping the whole Article card? Now it is taking only Image. 

Do we need testID?